File: MPI/mpicheck/tests/functions/1.0/MPI_Bcast.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    void *var_0;
    int var_1;
    MPI_Datatype var_2;
    int var_3;
    MPI_Comm var_4;
    int ret;
    /* calls */
    ret = MPI_Bcast(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Bcast(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Bcast.f

        
        program main
        include 'mpif.h'
        
        TYPE(INTEGER) var_0(10)
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_bcast(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_bcast(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    